pvlib/pvsystem.py

```diff
@@ -944,75 +945,61 @@ def physicaliam(aoi, n=1.526, K=4., L=0.002):
-    model to determine IV curves at irradiance = S, cell temperature =
-    Tcell.
+    Calculates five parameter values for the single diode equation at 
+    effective irradiance and cell temperature using the De Soto et al. 
+    model described in [1]. The five values returned by calcparams_desoto
+    can be used by singlediode to calculate an IV curve.
 
     Parameters
     ----------
-    poa_global : numeric
-        The irradiance (in W/m^2) absorbed by the module.
+    effective_irradiance : numeric
+        The irradiance (W/m2) that is converted to photocurrent.
 
     temp_cell : numeric
         The average cell temperature of cells within a module in C.
 
-    alpha_isc : float
+    alpha_sc : float
         The short-circuit current temperature coefficient of the
-        module in units of 1/C.
-
-    module_parameters : dict
-        Parameters describing PV module performance at reference
-        conditions according to DeSoto's paper. Parameters may be
-        generated or found by lookup. For ease of use,
-        retrieve_sam can automatically generate a dict based on the
-        most recent SAM CEC module
-        database. The module_parameters dict must contain the
-        following 5 fields:
-
-            * a_ref - modified diode ideality factor parameter at
-              reference conditions (units of eV), a_ref can be calculated
-              from the usual diode ideality factor (n),
-              number of cells in series (Ns),
-              and cell temperature (Tcell) per equation (2) in [1].
-            * I_L_ref - Light-generated current (or photocurrent)
-              in amperes at reference conditions. This value is referred to
-              as Iph in some literature.
-            * I_o_ref - diode reverse saturation current in amperes,
-              under reference conditions.
-            * R_sh_ref - shunt resistance under reference conditions (ohms).
-            * R_s - series resistance under reference conditions (ohms).
+        module in units of A/C.
+
+    a_ref : float
+        The product of the usual diode ideality factor (n, unitless), 
+        number of cells in series (Ns), and cell thermal voltage at reference
+        conditions, in units of V.
+
+    I_L_ref : float
+        The light-generated current (or photocurrent) at reference conditions,
+        in amperes.
+        
+    I_o_ref : float
+        The dark or diode reverse saturation current at reference conditions,
+        in amperes.
+        
+    R_sh_ref : float
+        The shunt resistance at reference conditions, in ohms.
+        
+    R_s : float
+        The series resistance at reference conditions, in ohms.
 
     EgRef : float
-        The energy bandgap at reference temperature (in eV).
-        1.121 eV for silicon. EgRef must be >0.
+        The energy bandgap at reference temperature in units of eV.
+        1.121 eV for silicon. EgRef must be >0.  For parameters read from
```

Comment:
nit: Might it be better to say crystalline Silicon here (as opposed to amorphous, etc.)?

Reply:
Good point, 1.121eV is for crystalline silicon.